docs/table/construct_table.rst

```diff
@@ -256,6 +256,21 @@ Every row must have the same set of keys or a ValueError will be thrown::
     ...
   ValueError: Row 0 has no value for column c
 
+You can also preserve the column order by using ``OrderedDict``. If the first item is an 
+``OrderedDict`` then the order is preserved:
+
+  >>> from collections import OrderedDict
+  >>> row1 = OrderedDict({'b': 1, 'a':0})
+  >>> row2 = OrderedDict({'b': 11, 'a':10})
+  >>> rows = [row1,row2]
+  >>> Table(rows=rows)  # doctest: +SKIP
```

Comment:
Please don't skip this test

Reply:
OK, I can remove the test skip. Question, I used the example above this to format my addition, should that example also not have a `# doctest: +SKIP` or is there a reason for that example?

Reply:
yes, please don't skip any examples unless there is a very good reason for it. Having the doctest run for them makes sure they keep working even as we change the code during the years.

Reply:
OK, changed with the latest commits